app/components/equipment_creation_component.rb

```diff
@@ -1,8 +1,9 @@
 class EquipmentCreationComponent < ViewComponent::Base
 
   def initialize(equipment:, source: )
```

Comment:
I would go for consistency across components with the parameters ordering for similar interfaces. For example, the equipment creation component has the source parameter first and then the equipment, while this one has the source parameter second. If you maintain consistency for similar interfaces then things can be more guessable without having to check the implementation: principle of least surprise. Does not matter too much here as they are keyword parameters so you have to be explicit when you use it anyway, but the definition does suggest an ordering to users, so a consistent ordering in the definition will lead to consistent usage, which should make the code easier to read.